playbooks/roles/minishift/tasks/main.yml

```diff
@@ -1,6 +1,10 @@
 ---
 # setup minishift
 
+- fail:
```

Comment:
setup_minishift=false means this would never run so I think this is unnecessary

Reply:
this is if `setup_minishift == true` **AND** someone forced a value for `openshift_cluster_ip`
This covers the case where we are deploying to UpShift. We do not need minishift installed right?

Reply:
Then this role never runs anyway so it wouldn't be an issue would it?
https://github.com/arilivigni/contra-env-setup/blob/master/playbooks/setup.yml#L11

Reply:
Discussed on IRC makes sense to keep this in place!